api/v1alpha/framework/cypress.schema.json

```
@@ -60,6 +60,27 @@
         },
         "version": {
           "$ref": "../subschema/common.schema.json#/definitions/version"
+        },
+        "reporters": {
+          "description": "Set of reporter to use.",
+          "type": "array",
+          "minimum": 0,
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "Name of the reporter.",
+                "type": "string",
+                "examples": [
+                  "dot",
+                  "nyan"
+                ]
+              },
+              "options": {
+                "description": "Options to pass to the reporter."
+              }
+            }
```

Would be helpful to restrict further unknown properties (helps users if they misspell attributes).

```suggestion
            },
            "additionalProperties": false
```